lib/main.js

```diff
@@ -327,6 +411,15 @@ function signText(text) {
       return ERROR_INTERNAL;
     }
 
+    // Get certificate slot
+    let slot = ctypes.cast(cert, CERTCertificateStr.ptr).contents.slot;
+    // Check provided password if cert slot needs login
+    if(PK11_CheckUserPassword(slot, certName.password) != SECSuccess) {
+      log("Token password verification failed");
```

Comment:
I've been referencing the name of the function in these log messages (in this case it would be 'PK11_CheckUserPassword'). I guess it doesn't matter either way as long as the error message points us to what call failed.